core/src/main/java/com/airbnb/aerosolve/core/models/AdditiveModel.java

```diff
@@ -294,4 +294,8 @@ public void updateDense(float gradWithLearningRate,
       }
     }
   }
+
+  public void resample(int newBins) {
```

Comment:
That's a fair point. The motivation of moving to core was to avoid java/scala conversion. I will move it back and use for loop instead.